xds/internal/resolver/serviceconfig.go

```diff
@@ -245,9 +245,7 @@ func (cs *configSelector) generateHash(rpcInfo iresolver.RPCInfo, hashPolicies [
 			generatedHash = true
 			generatedPolicyHash = true
 		case xdsresource.HashPolicyTypeChannelID:
-			// Hash the ClientConn pointer which logically uniquely
-			// identifies the client.
-			policyHash = xxhash.Sum64String(fmt.Sprintf("%p", &cs.r.cc))
+			policyHash = cs.r.clientID
```

Comment:
Update HashPolicyTypeChannelID's comment please.

Reply:
Done; PTAL

Reply:
I meant in xdsresource.

Reply:
Whoops. Done.